Library/Formula/sfnt2woff.rb

```diff
@@ -0,0 +1,18 @@
+class Sfnt2woff < Formula
+  desc "Convert TTF and OTF fonts to WOFF format."
+  homepage "https://people.mozilla.org/~jkew/woff/"
+  url "https://people.mozilla.org/~jkew/woff/woff-code-latest.zip"
```

Comment:
We can't use a `latest` zip because it'll change underneath us without warning. We'll need an actual versioned release to include this I'm afraid.